libs/ECS/Components/WeaknessComponent.hpp

```diff
@@ -0,0 +1,36 @@
+/*
+** EPITECH PROJECT, 2022
+** Project
+** File description:
+** WeaknessComponent
+*/
+
+#ifndef WEAKNESSCOMPONENT_HPP_
+#define WEAKNESSCOMPONENT_HPP_
+
+#include "Component.hpp"
+
+namespace ecs
+{
+
+    /// @brief The Weakness component, to create a target for projectiles
+    class Weakness : public Component {
+      public:
+        /// @brief The abscissa of the Weakness
+        int x;
```

Comment:
Weakness is only a point on an entity ?
A circle (Center + radius) can be more adapted.

Reply:
I agree, it will be easier with the maths later